src/main/java/org/fundacionjala/at15/katas/pokerhands/diana/HighCard.java

```diff
@@ -0,0 +1,16 @@
+package org.fundacionjala.at15.katas.pokerhands.diana;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class HighCard extends CardAnalysis {
+    private static final int ONE = 1;
+    protected int range;
```

Comment:
if all subclasses have this attribute, you should consider to move it to the super class. That way, you don't need to defined on each subclass you create